README.md

```
@@ -1,247 +1,132 @@
+cat examples/bullish_aapl.json | intraday-analyzer
+
+# Or pass a file path
+intraday-analyzer examples/bullish_aapl.json
+
+# Run tests
+pytest
+```
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)